core/workspace_svg.js

```diff
@@ -1372,19 +1322,20 @@ Blockly.WorkspaceSvg.prototype.showContextMenu_ = function(e) {
     Blockly.Events.setGroup(false);
   }
 
+  var deleteCount = deleteList.length;
   var deleteOption = {
-    text: deleteList.length == 1 ? Blockly.Msg['DELETE_BLOCK'] :
-        Blockly.Msg['DELETE_X_BLOCKS'].replace('%1', String(deleteList.length)),
-    enabled: deleteList.length > 0,
+    text: deleteCount == 1 ? Blockly.Msg['DELETE_BLOCK'] :
```

Comment:
Switch this to use Blockly.ContextMenu.createOption, like all the others (since you've switched all the others)?